CHANGELOG.md

```diff
@@ -55,8 +55,24 @@ All notable changes to this project will be documented in this file.
         * Initial support
         * Very basic at this point and subject to change
         * Looking for any testing and/or feedback
+        * `get_channel_name_from_string()`
     * #### Changed
+        *  New enums:
+
+           | Enum name                  | Where used                                                                       |
+           |----------------------------|----------------------------------------------------------------------------------|
+           | `DigitalEdge`              | `digital_edge_conditional_jump_trigger_edge`, `digital_edge_start_trigger_edge`  |
+           | `ApertureTimeUnits`        | `ppmu_aperture_time_units`, `ppmu_configure_aperture_time(units)`                |
+           | `PpmuOutputFunction`       | `ppmu_output_function`                                                           |
```

Comment:
PPMU is an acronym, shouldn't we spell this PPMUOutputFunction? Or how do we treat other acronyms when they are in the beginning of a type?

Reply:
From nidcpower `DCNoiseRejection` and from nidmm `ADCCalibration`

Change to PPMU*